### ISSUE TITLE
A warning during training

### ISSUE THREAD

**Issue by TessGreymane** [salman-h-khan/PL-ZSD_Release#9]
I faced a warning during training this code on coco2014.

`train_vocab_w2v.py:367: UserWarning: Output "nms" missing from loss dictionary. We assume this was done on purpose, and we will not be expecting any data to be passed to "nms" during training.
  optimizer=keras.optimizers.adam(lr=1e-5, clipnorm=0.001)
202
209`

But this warning did not affect the running process. I want to know whether this warning will affect the performance of trained models. If so, how can I avoid this warning during my training process? I appreciate that if you can help me to solve this problem. Thank you for your attention.

**Comment by shafin5** [comment]
You can ignore this warning. It should not affect performance.
